### DISCUSSION TITLE
HmIP-HAP/HmIPW-DRAP Support Patch  nach Update auf Home Assistant OS 12.2 keine Funktion mehr  (Addon RasberryMatic)

### DISCUSSION THREAD

**Discussion by horstvogel** [OpenCCU/OpenCCU#2714] [🇩🇪 Fragen&Antworten] [answered]
Moin, nach dem Update auf 
auf Home Assistant 12.2 werden die HmIP- HAP zwar noch angezeigt, aber es erfolgt anscheinend kein Datenverkehr mehr über die HmIP-HA. 

Kann das jemand bestätigen. Natürlich habe ich den Patch wie immer nach dem Neustart wieder ausgeführt...

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#hmip-haphmipw-drap-support-patch

HAP werden erkannt, es geht aber kein Verkehr über die "Repeater" 

![image](https://github.com/jens-maus/RaspberryMatic/assets/25108737/e802a31a-b41c-4966-8a4e-42fddfdbddc4)

**Comment by horstvogel** [comment, marked as answer]
Problem hat sich erledigt, warum auch immer geht das nun wieder.

**Comment by renelinne** [comment]
Ich habe allerdings das selbe Problem nach dem Update.
Beim Ausführen des Patches kommen folgende Meldungen:
![Screenshot 2024-04-12 181319](https://github.com/jens-maus/RaspberryMatic/assets/116603812/9b6870c8-8bb2-4da5-bdf2-8c9902a88c6d)

**Reply by Baxxy13** [reply]
Was passiert denn wenn du bei "Main Subnet" mal der "Empfehlung" folgst und 192.168.178.0/24 eingibst?

**Reply by jens-maus** [reply]
Du hast ja auch an der zweiten Eingabe kein subnetz eingegeben sondern die ip Adresse mit /24 hinten. Da ist klar das es da zu einer Fehlermeldung kommt.

**Reply by renelinne** [reply]
Asche auf mein Haupt.... 
Aber irgend etwas ist anders, das ich in der Vergangenheit an den ersten drei Prompts immer nur mit Enter durchgeskippt hatte und erst beim Kontrainernamen und der individuellen IP eingaben gemacht hatte.
Danke für den Hinweis. Es funktioniert!
